odpc.client/src/composables/use-form-validator.ts

```diff
@@ -0,0 +1,100 @@
+import { ref, onUnmounted, type ComputedRef, watchEffect } from "vue";
+
+export const useFormValidator = (formRef: ComputedRef<HTMLFormElement | undefined>) => {
```

Comment:
als je dit doet hoef je in de aanroep geen computed() te gebruiken:
```suggestion
export const useFormValidator = (formRef: Readonly<Ref<HTMLFormElement | undefined>>) => {
```

Reply:
Anders opgelost, formRef exposed vanuit de composable,

Reply:
gaaf!